CHANGELOG-1.2.md

```diff
@@ -0,0 +1,10 @@
+# Changelog since v1.1.0
+
+## Deprecations
+
+## Notable Features
+
+* Support attachment of inline volumes migrated to CSI [#153](https://github.com/kubernetes-csi/external-attacher/pull/154)
+* Action required: PATCH is used instead of POST to modify VolumeAttachments on API servers. Please note changes in the external attacher RBAC policy. [#149](https://github.com/kubernetes-csi/external-attacher/pull/149)
+
+## Other notable changes
```

Comment:
There were some other changes that I think are worth including too. We should use the release-note tool starting from commit e157b6b514ace191f46d4c879491cb7d11684659.

Reply:
Current instructions are here: https://github.com/kubernetes-csi/docs/issues/142#issuecomment-499954471